### PR TITLE
Update contact form url on /security/fips 

### PR DESCRIPTION
**Pull request by minkyngkm**
## Done

- Updated connected forms so the two different forms are used for the `Contact us` button and ` Join our free beta programme for a FIPS-validated OpenJDK `link each. 

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Please refer to [form doc](https://docs.google.com/document/d/1dA6y-0VCsug50cUnWsNsFQYp1-leg8JZ_EU1rCKJeiA/edit#) and [brief](https://docs.google.com/document/d/1dzUObyMPCrlLRNVj7lfjxIG02os1tTUDAxN3IB0UNLo/edit) if needed
1. Go to `/security/fips` [Demo](https://ubuntu-com-10765.demos.haus/security/fips)
2. Click `Contact us` and check if it a modal form shows
3. Click `Join our free beta programme for a FIPS-validated OpenJDK` and check if it goes to `/security/fips/contact-us`

### CODE REVIEW

**Comment by bethcollins92** [comment]
As discussed, at the moment it isn't possible to have 2 different contact modals on the same page due to the way the javascript currently works. If you inspect the page and select the `Join our free beta programme for a FIPS-validated OpenJDK` you'll see the actual element that highlights in the console is the `contact us` one as this is the one with the `contact-form-container` id

**Comment by minkyngkm** [comment]
> As discussed, at the moment it isn't possible to have 2 different contact modals on the same page due to the way the javascript currently works. If you inspect the page and select the `Join our free beta programme for a FIPS-validated OpenJDK` you'll see the actual element that highlights in the console is the `contact us` one as this is the one with the `contact-form-container` id

Thanks @bethcollins92! I moved the form that was connected to `Join our free beta programme for a FIPS-validated OpenJDK` on `/security/fips/contact-us` as PM requested. It's not a modal form any more. Could you please have a look if its okay? :)

**Comment by bethcollins92** [comment]
@pmahnke - The PM has asked for 2 different interactive contact forms to be on the same page which we can't due to the current way the javascript works. Min has made a new `/contact-us` page with all the quesitons - I know this isn't convention but is this ok?

File: templates/shared/_security-fips.html [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+					</div>
+				</fieldset>
+				<fieldset class="u-no-margin--bottom">
+					<h3 class="p-heading--five">Do you use any of the packages below?</h3>
+					<div class="row u-no-padding">
+						<div class="col-6">
+							<input type="checkbox" id="security-fips-tomcat">
+							<label for="security-fips-tomcat">tomcat</label>
+							<input type="checkbox" id="security-fips-other">
+							<label for="security-fips-other">Other</label>
```
Comment (bethcollins92) [inline templates/shared/_security-fips.html:35]: The other checkbox normally has an input field for the users to type. Take a look at https://ubuntu.com/openstack#get-in-touch for example

**Comment by nmav** [comment]
Note that there are no two contact forms in the FIPS page, there is one contact form (which is currently broken), and an invitation to register for a beta testing for java campaign link.

**Comment by minkyngkm** [comment]
> Note that this is not a request to have two contact forms in the FIPS page. The request is to change no contact forms in the fips page (there is one contact form which is currently broken), and add an invitation to register for a beta testing for java campaign on top of the page.

Hi @nmav! Working on this now. It will be updated soon!

**Comment by nmav** [comment]
Seeing the form I'd have expected the opposite. The temporary form (Java) to be on a separate page, and the permanent (Contact us) being on the modal. The reason is that in one, two months the java campaign will finish and we remove this form so it would make sense for contact-us to still be the modal one

**Comment by nmav** [comment]
LGTM, though the thank you page at the end is very awkward. It shows "Contact us about LTS Docker Images"

File: templates/shared/_security-fips.html [hunk 2]
```diff
@@ -0,0 +1,226 @@
...
+              <div class="js-other-container">
+                <input type="checkbox" class="js-other-container__checkbox" id="security-fips-other">
+                <label for="security-fips-other">Other</label>
+                <input type="text" id="security-fips-other-specified" class="js-other-container__input" placeholder="Please specify" style="margin-top: .25rem; visibility: hidden" aria-label="Other workloads">
+              </div>
+            </div>
+					</div>
+				</fieldset>
+				<fieldset class="u-no-margin--bottom fieldset">
+					<h3 class="p-heading--five">Which java runtime do your applications depend on?</h3>
```
Comment (mtruj013) [inline templates/shared/_security-fips.html:43]: Just a typo in the copy doc, Java should be capitalized here 
```suggestion
					<h3 class="p-heading--five">Which Java runtime do your applications depend on?</h3>
```

**Review by mtruj013 — APPROVED** [review]

**Pull request merged**